topology/probes/netlink/netlink.go

```diff
@@ -514,12 +514,9 @@ func (u *NetNsProbe) getRoutingTables(link netlink.Link, table int) *RoutingTabl
 			routingTable = &RoutingTable{ID: int64(r.Table), Src: r.Src}
 		}
 
-		protocol, prefix := int64(r.Protocol), ""
```

Comment:
this check should remain otherwise the agent will crash

Reply:
I fixed this issue and some other. I think that should work now for IPV4 and IPV6
https://paste.fedoraproject.org/paste/6~UNzHnW8gFEfqWPPtoMoA